aopy/visualization.py

```diff
@@ -445,6 +445,73 @@ def plot_targets(target_positions, target_radius, bounds=None, alpha=0.5, origin
             ax.set_aspect('equal', adjustable='box')
     if bounds is not None: set_bounds(bounds, ax)
 
+def plot_circles(circle_positions, circle_radius, circle_color = 'b', bounds=None, alpha=0.5, origin=(0, 0, 0), ax=None, unique_only=True):    
+    '''
+    Add targets to an axis. If any targets are at the origin, they will appear 
+    in a different color (magenta). Works for 2D and 3D axes
```

Comment:
This docstring is out of date

Reply:
will create issue to reflect outdated docs in parent/child functions

Reply:
i fixed it.